vizro-ai/docs/pages/user_guides/run_vizro_ai.md

```diff
@@ -71,15 +71,12 @@ Vizro-AI's `_get_chart_code` method returns a dictionary containing the Python c
         vizro_ai = VizroAI()
 
         df = px.data.gapminder()
-        output = vizro_ai._get_chart_code(df, "describe life expectancy per continent over time", explain=True)
-
-        code_string = output.get("code_string")
-        code_explanation = output.get("code_explanation")
-        business_insights = output.get("business_insights")
+        code_string = vizro_ai._get_chart_code(df, "describe life expectancy per continent over time")
         ```
     === "code_string"
         [![ResultCode]][ResultCode]
 
     [ResultCode]: ../../assets/user_guides/code_string_app_integration.png
 
 The returned `code_string` can be used to dynamically render charts within your application. You may have the option to encapsulate the chart within a `fig` object or convert the figure into a JSON string for further integration.
+In case you would like to use the insights or code explanation, you can use `vizro_ai._run_plot_tasks(df, ..., explain=False)`, which returns a dictionary containing the code explanation and chart insights alongside the code.
```

Comment:
```suggestion
In case you would like to use the insights or code explanation, you can use `vizro_ai._run_plot_tasks(df, ..., explain=True)`, which returns a dictionary containing the code explanation and chart insights alongside the code.
```